lib/Doctrine/ORM/Tools/Console/Command/MappingDescribeCommand.php

```diff
@@ -257,7 +257,7 @@ private function formatMappings(array $propertyMappings): array
         foreach ($propertyMappings as $propertyName => $mapping) {
             $output[] = $this->formatField(sprintf('  %s', $propertyName), '');
 
-            foreach ($mapping as $field => $value) {
+            foreach ((array) $mapping as $field => $value) {
```

Comment:
I assume having an IteratorAggregate would be too much and is supposed to be kept simple.

Reply:
Yes, and it's not like this is a collection or something.